selfdrive/hardware/base.h

```diff
@@ -4,11 +4,18 @@
 #include <fstream>
 
 
+enum HardwareDeviceType {
```

Comment:
We already have a definition for device type: https://github.com/commaai/cereal/blob/master/log.capnp#L44